### ISSUE TITLE
Comment your public methods

### ISSUE THREAD

**Issue by chrisjenx** [jfeinstein10/SlidingMenu#237]
Please do some developer UX on your code. I have been going though tweaking etc. and struggling in places, doing guess work for the best part.

**Comment by jfeinstein10** [comment]
Where exactly were you looking? I know that everything that's Dev facing in the SlidingMenu class has full javadoc comments, but other places don't. I'll try to fix that.

**Comment by chrisjenx** [comment]
CustomView Behind/Above I have worked out most of them but yeah, either
slightly better method names or javadoc would be amazing.

Cheers,
Chris

**Comment by jfeinstein10** [comment]
Ah, yeah that's a gray area haha. What were you trying to modify? if you don't mind me asking

**Comment by chrisjenx** [comment]
Yeah sure, I'm modifying it to do top/bottom kinda not SlidingMenu more
SlidingPanel but still going to look cool :)

**Comment by chrisjenx** [comment]
Doing a bit of refactoring, I'll do a pull request but it might not be
worth looking though the code and implementing yourself so you can make
tweaks along the way.

**Comment by jfeinstein10** [comment]
Oh sweetness! Yeah, that's going to be a bit of a pain to implement if you want combinations (i.e. left and top)
Still lots of potential for awesome :+1:
